test/suite/csprojReader.test.ts

```diff
@@ -0,0 +1,194 @@
+import * as assert from 'assert';
+import * as path from 'path';
+import * as fs from 'fs';
+import CsprojReader from '../../src/project/csprojReader';
+
+const fixture_path= path.resolve(__dirname, '../suite/');
+interface Fixture {
+    filename: string,
+    csproj : string,
+    expected : string | undefined,
+}
+
+suite('CsprojReader', () => {
+    const validTargetFramework : Array<string> = [
+        'netcoreapp1.0',
+        'netcoreapp1.1',
+        'netcoreapp2.0',
+        'netcoreapp2.1',
+        'netcoreapp2.2',
+        'netcoreapp3.0',
+        'netcoreapp3.1',
+        'net5.0',
+        'net6.0',
+    ];
+
+    const rootNameSpacefixtures : Array<Fixture> = [
+        {
+            filename: 'xamarin.csproj',
+            csproj: `
+            <Project Sdk="Microsoft.NET.Sdk">
+                <PropertyGroup>
+                    <RootNamespace>Xamarin.Forms</RootNamespace>
+                </PropertyGroup>
+            </Project>`, 
+            expected: 'Xamarin.Forms',
+        },
+        {
+            filename: 'linq.csproj',
+            csproj: `<Project Sdk="Microsoft.NET.Sdk">
+                        <PropertyGroup></PropertyGroup>
+                        <PropertyGroup></PropertyGroup>
+                        <PropertyGroup>
+                            <RootNamespace>System.Linq</RootNamespace>
+                        </PropertyGroup>
+                    </Project>`, 
+            expected: 'System.Linq',
+        },
+        {
+            filename: 'empty-group.csporj',
```

Comment:
Doesn't really matter for now, but there are a few `.csporj` filenames. This will matter in the future, when the [todo in csprojReader](https://github.com/KreativJos/csharpextensions/blob/7fb9947530a75f4fb134de6d7e986d1abb750322/src/project/csprojReader.ts#L18) is fixed.